GOVERNANCE.md

```diff
@@ -136,26 +175,50 @@ issue, but must recuse themselves from voting on the issue.
-* If more than one candidate has the same highest median-grade, the MJ winner is discovered by removing (one-by-one) any grades equal in value to the shared median grade from each tied candidate's total. This is repeated until only one of the previously tied candidates is currently found to have the highest median-grade.
-* If ties are still present after this second round, the winner will be chosen at random. Each person tied will pick a number between 0 and 100, and a random integer will be generated from random.org. The person with the closet circular distance will be selected.
-* At the conclusion of voting, all the results will be posted.
-* The decision about who can vote after the first election is deferred to the council. See below for details
+* Voting occurs over a period of at least 1 week, at the conclusion of the nominations.
+  Voting is blind and mediated by either an application or a third party like NumFOCUS.
+  Each voter can vote zero or more times, once per each candidate. As this is not about
+  ranking but about capabilities, voters vote on a yes/neutral/no basis
+  per candidate -- “would I trust this person to lead ArviZ?”.
+* Candidates are evaluated independently, each candidate having 60% or more of
+  yes votes _and_ less or equal than 20% of no votes is chosen. If the number
+  of chosen candidates is >=4 and <=10 all candidates are confirmed and the
+  election process stops here.
+* In the event that either not enough or too many candidates were confirmed,
+  candidates are ranked by interpreting yes=+1, neutral=0 and no=-1. If too
+  many candidates were confirmed, the 10 candidates with higher rank are
+  elected. If not enough candidates were chosen, the 4 candidates with higher
+  rank are elected.
+* In the event of a tie there will be a runoff election for the tied candidates. To avoid
+  further ties and discriminate more among the tied candidates, this vote will be held
+  by [Majority Judgment](https://en.wikipedia.org/wiki/Majority_judgment) (MJ): for each
+  candidate, voters judge their suitability for office as either Excellent, Very Good,
+  Good, Acceptable, Poor, or Reject. Multiple candidates may be given the same grade by
+  a voter. The candidate with the highest median grade is the winner.
+* If more than one candidate has the same highest median-grade, the MJ winner is
+  discovered by removing (one-by-one) any grades equal in value to the shared median
+  grade from each tied candidate's total. This is repeated until only one of the
+  previously tied candidates is currently found to have the highest median-grade.
+* If ties are still present after this second round, the winner will be chosen at random.
+  Each person tied will pick a number between 0 and 100, and a random integer will be
+  generated from random.org. The person with the closest circular distance will be selected.
```

Comment:
Who picks the integer? How will this not be easy to game?

Reply:
Define circular distance?

Reply:
Explicitly state the random integer will be in [0,100]?

Reply:
I would say the same person (aka numfocus representative) who handled the other two previous voting does. I think a difficult to game process would be all candidates send their number to numfocus and they generate the number. The process is repeated as many times as necessary (the same number may be chosen leading to yet another tie).

About circular distance, how about `min(|𝑎−𝑏|,100−|𝑎−𝑏|)` (taken from https://math.stackexchange.com/questions/1951658/calculate-distance-between-two-values-in-a-cycle) and have generated numbers be in [0, 99] (or use 101 and [0, 100])?

Reply:
How about [1,100] (which is the default for the front page generator at random.org) and `min(|𝑎−𝑏|,100−|𝑎−𝑏|)`? This seems to do the right thing.